website/src/_posts/2019-04-19-beam-2.12.0.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Apache Beam 2.12.0"
+date:   2019-04-19 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+        - apilloud
+
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+We are happy to present the new 2.12.0 release of Beam. This release includes both improvements and new functionality.
+See the [download page]({{ site.baseurl }}/get-started/downloads/#2120-2019-04-19) for this release.<!--more-->
+For more information on changes in 2.12.0, check out the
+[detailed release notes](https://jira.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12344944).
+
+## Highlights
+
+### I/Os
+
+* Implement a BigQuery custom sink for Python SDK.
+* Add support to specify a query in CassandraIO for Java SDK.
+
+### New Features / Improvements
+
+* Enable Bundle Finalization in Python SDK.
```

Comment:
`Enable Bundle Finalization in Python SDK.` -> `Enable Bundle Finalization in Python SDK for portable runners.`